kroxylicious-multitenant/src/test/java/io/kroxylicious/proxy/filter/multitenant/MultiTenantTransformationFilterTest.java

```diff
@@ -204,12 +216,12 @@ public void beforeEach() {
     }
 
     public static Stream<Arguments> requests() throws Exception {
-        return requestResponseTestDefinitions().map(t -> Arguments.of(t.source(), t.apiKey(), t.header(), t.request()));
+        return requestResponseTestDefinitions().map(t -> Arguments.of(t.testName(), t.apiKey(), t.header(), t.request()));
     }
 
-    @ParameterizedTest
+    @ParameterizedTest(name = "{0}")
     @MethodSource
-    void requests(@SuppressWarnings("unused") String source, ApiMessageType apiMessageType, RequestHeaderData header, ApiMessageTestDef requestTestDef) {
+    void requests(@SuppressWarnings("unused") String testName, ApiMessageType apiMessageType, RequestHeaderData header, ApiMessageTestDef requestTestDef) {
```

Comment:
nit: `requests` -> `requestsTransformed`

Though I'd be tempted to leave `Stream<Arguments> requests()` and use `@MethodSource("requests")` here